tests/test_instanceconnectionmanager_unit.py

```diff
@@ -46,3 +47,68 @@ def test_InstanceConnectionManager_wrong_connection_string():
     loop = asyncio.new_event_loop()
     with pytest.raises(CloudSQLConnectionError):
         InstanceConnectionManager("test-project:test-region", loop)
+
+
+def test_InstanceConnectionManager_get_ephemeral():
```

Comment:
Ideally this test would mock the response somehow, and we would have some system tests somewhere that would test things end to end.

Reply:
I'll make the changes to these tests in another PR, but it will be done